### ISSUE TITLE
Keyboard navigation to rename bookmarks does not work

### ISSUE THREAD

**Issue by Piiit** [linuxmint/nemo#1515]
*How to reproduce?*
- Open nemo, select a file or folder inside the main window
- Press F6, you can now navigate bookmarks with arrow keys
- Press F2 to rename a bookmark (It renames the file inside the main window instead)

The same thing works with mouse:
- click on a bookmark, 
- press F2
- rename it.....

**Comment by icarter09** [comment]
Nemo v4.4.2
Mint 19.3 64bit

Verified this issue still exists.

**Comment by icarter09** [comment]
Nemo 4.6.5
Mint 20 64bit

I'm seeing that F2 does work, but not directly. 

Steps to reproduce:
1.  If the folder location selected in the sidebar does not have any elements within it, then F2 works to rename that folder.
2.  If the folder location selected in the sidebar DOES have elements within it, then F2 does not immediately work. You first click in the window, then click back again on the folder in the sidebar. F2 works to rename it. 

The issue looks to be that focus first goes to the window displaying elements of a given folder.
